README.md

```diff
@@ -287,7 +287,7 @@ Metastore table options:
 | `name`                       | Name of the metastore table                                                       |
 | `format`                     | Storage format (`parquet` or `delta`)                                             |
 | `path`                       | Path to the data in the metastore.                                                |
-| `table`                      | DataLake table name (if DataLake tables are the underlying storage).              |
+| `table`                      | Delta Lake table name (if Delta Lake tables are the underlying storage).          |
```

Comment:
this was a bit confusing, since I think in this particular reference "table" if in reference to Delta Lake table, rather than generic concept of a table in a generic data lake

could be wrong?